### PR TITLE
added links to alternative ways to install Friendica to the docs

### PR DESCRIPTION
**Pull request by tobiasd**

### CODE REVIEW

File: INSTALL.txt [hunk 1]
```diff
@@ -56,6 +56,13 @@ you wish to communicate with the Diaspora network.
 	- For alternative server configurations (such as Nginx server and MariaDB
 	database engine), refer to the wiki at https://github.com/friendica/friendica/wiki
 
+This guide will walk you through the manual installation process of Friendica.
+If this is nothing for you, you might be interested in
+
+* the Friendica Docker image (https://github.com/friendica/docker) or
+* how install Friendica with YunoHost] https://github.com/YunoHost-Apps/friendica_ynh).
```
Comment (annando) [inline INSTALL.txt]: What is the ```]``` doing there?

Comment (tobiasd) [reply INSTALL.txt]: Leftover from C&P operations from the Markdown file.

File: INSTALL.txt [hunk 2]
```diff
@@ -56,6 +56,13 @@ you wish to communicate with the Diaspora network.
 	- For alternative server configurations (such as Nginx server and MariaDB
 	database engine), refer to the wiki at https://github.com/friendica/friendica/wiki
 
+This guide will walk you through the manual installation process of Friendica.
+If this is nothing for you, you might be interested in
```
Comment (MrPetovan) [inline INSTALL.txt]: What does "If this is nothing for you" mean?

**Comment by tobiasd** [comment]
> What does "If this is nothing for you" mean?

DeepL has the opinion that I should have used "not" instead of "nothing". So it is denglish it seems.

**Comment by MrPetovan** [comment]
@annando Next time please let a real human with actual english knowledge look at documentation changes!

**Comment by tobiasd** [comment]
> @annando Next time please let a real human with actual english
> knowledge look at documentation changes!

Could you please stop de-humanizing other contributors to the project.

**Comment by annando** [comment]
I guess he meant a "near natural speaker". Although I should have noticed it as well.

**Comment by MrPetovan** [comment]
Let's do an "English copy approval" and a "German copy approval". Once we have both we can merge.

**Comment by MrPetovan** [comment]
> @annando is as much human as you are.

Did anyone ever check?

**Comment by annando** [comment]
Not sure if I'm human, but you seemed to be.

**Comment by MrPetovan** [comment]
I wouldn't jump to conclusions too hastily.

**Pull request merged**